src/datachain/query/schema.py

```diff
@@ -19,6 +19,17 @@
 DEFAULT_DELIMITER = "__"
 
 
+def file_signals(row, signal_name="file"):
```

Comment:
Workaround to make old code work with new listings, until we decide what do do with these classes in the first place